sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -6,11 +6,26 @@
 import requests
 from SDSCode.database import database as db
 from SDSCode.database.models import Base
-from SDSCode.dependency_config import downstream_dependents, upstream_dependents
-from SDSCode.dependency_config_ultra import (
+from SDSCode.dependency.dependency_config import (
+    downstream_dependents,
+    upstream_dependents,
+)
+from SDSCode.dependency.dependency_config_hi import (
+    downstream_dependents as downstream_dependents_hi,
+)
```

Comment:
I'll just reiterate my personal preference to avoid so many modules and imports. There are a lot of imports going on here. I find it easier to read if you only import one level and then traverse via attribute getting later.

```suggestion
from SDSCode import dependency

# Then use the nested things as necessary
dependency.config.downstream_dependents
dependency.config.upstream_dependents_hi
```

Also, you put `dependency.dependency_config_hi`, to me that would be easier to read as `dependency.config_hi` avoiding the repetition.